frontend/src/hooks/useEmployees.tsx

```diff
@@ -28,9 +28,17 @@ export default function useEmployees() {
             .catch(console.error)
     }
 
+    function deleteEmployee(id:string) {
+        return axios.delete("/api/employees/" +id)
+            .then(() => {
+            loadAllEmployees()
```

Comment:
Maybe you could optimize this method a little bit, so that you don't have to load all employees again.